composer.json

```diff
@@ -20,18 +20,34 @@
       "name": "Carlos Escribano",
       "email": "carlos@doofinder.com",
       "role": "Developer"
+    },
+    {
+      "name": "Domingo Arteaga",
+      "email": "domingo@doofinder.com",
+      "role": "Developer"
+    },
+    {
+      "name": "Swagger and contributors",
+      "homepage": "https://github.com/swagger-api/swagger-codegen"
     }
   ],
   "require": {
-    "php": ">=5.3.0"
+    "php": ">=5.5 <=7.1",
```

Comment:
Why <= 7.1? Newest Magento versions are working on ~7.2 which means they won't be able to install module